docs/website/docs/developers/general/release-management.md

```diff
@@ -43,9 +43,15 @@ When selecting a candidate we use the following criteria:
 3. LLVM submodule commit ideally exists upstream (no cherry picks or patches)
 4. Includes packages for all platforms, including macOS and Windows
 
+<<<<<<< HEAD
 When you've identified a potential candidate, comment on the tracking issue with
 the proposal and solicit feedback. People may point out known regressions or
 request that some feature make the cut.
+=======
+When you've identified a potential candidate, email the
+iree-technical-discussion list with the proposal and solicit feedback. People
+may point out known regressions or request that some feature make the cut.
+>>>>>>> 5e939d8a8 (Switch to LF AI & Data mailing lists)
```

Comment:
Merge conflict markers in here.

Wait... pre-commit flagged that as _markdownlint_, not "check for merge conflicts"?? https://github.com/iree-org/iree/actions/runs/10710092192/job/29696125923?pr=18401

Reply:
Yes, by accident. Fixed that.